bakerydemo/base/blocks.py

```diff
@@ -77,3 +94,13 @@ class BaseStreamBlock(StreamBlock):
         icon="media",
         template="blocks/embed_block.html",
     )
+    details_block = DetailsBlock()
+
+
+# StreamBlocks for Details
```

Comment:
This is probably not needed, you can use a single block easily with StreamField

Reply:
So you suggest not adding the DetailsBlock option to the "old" `BaseStreamBlock` , just to the "new" `DetailsStreamBlock` that I created?